### ISSUE TITLE
About step2 training action Learning rate scheduler(lr_scheduler)

### ISSUE THREAD

**Issue by XuWuLingYu** [roboterax/video-prediction-policy#14]
In `step2_train_action_calvin.py`, the code shows:
```python
opt = model.configure_optimizers()["optimizer"]
Ir_scheduler = model.configure_optimizers()["lr_scheduler"]["scheduler"]
```
However, when I debug training, found that the learning rate during the whole training progress is not changing. When I update the code to:
```python
optim_dict = model.configure_optimizers()
opt = optim_dict["optimizer"]
Ir_scheduler = optim_dict["lr_scheduler"]["scheduler"]
```
The learning rate scheduler really worked. So I wonder if it is what you want, or just no lr scheduler in action training?
